src/lib/models/scheduled-notification.js

```diff
@@ -4,19 +4,56 @@ const Model = require('./model');
 
 const validators = require('./validators');
 
-const MESSAGE_STATUSES = {
+const messageStatuses = {
```

Comment:
These are not mappers, rather lists of constants defining what values various fields can take.  This is to satisfy Sonarcloud etc which doesn't generally suggests defining constants rather than repeating strings